### PR TITLE
Add workflow linter for delta live tables task

### LINKED ISSUE #1638: Workflow linter to load and lint code of Delta Live Table

### PR DESCRIPTION
**Pull request by pritishpai**
Closes #1638

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/jobs.py [hunk 1]
```diff
@@ -160,8 +160,14 @@ def _register_run_job_task(self, graph: DependencyGraph):  # pylint: disable=unu
     def _register_pipeline_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
         if not self._task.pipeline_task:
             return
-        # TODO: https://github.com/databrickslabs/ucx/issues/1638
-        yield DependencyProblem('not-yet-implemented', 'Pipeline task is not yet implemented')
+
+        pipeline = self._ws.pipelines.get(self._task.pipeline_task.pipeline_id)
+        pipeline_libraries = pipeline.spec.libraries
+        for library in pipeline_libraries:
+            if library.notebook:
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/jobs.py:167]: What about the wheel library? Also log the cases you've skipped

**Review by nfx — APPROVED** [review]
Few nits, otherwise lgtm

File: src/databricks/labs/ucx/source_code/jobs.py [hunk 2]
```diff
@@ -160,8 +160,14 @@ def _register_run_job_task(self, graph: DependencyGraph):  # pylint: disable=unu
     def _register_pipeline_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/jobs.py:160]:
```suggestion
    def _register_pipeline_task(self, graph: DependencyGraph):
```

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
Small change

File: src/databricks/labs/ucx/source_code/jobs.py (hunk 1 above)
Comment (pritishpai) [reply src/databricks/labs/ucx/source_code/jobs.py:167]: Pipeline library has only these 4 options:
https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/service/pipelines.py#L1234

**Pull request merged**